libraries/AP_DroneCAN/AP_DroneCAN.cpp

```diff
@@ -1457,13 +1457,41 @@ bool AP_DroneCAN::is_esc_data_index_valid(const uint8_t index) {
 void AP_DroneCAN::handle_debug(const CanardRxTransfer& transfer, const uavcan_protocol_debug_LogMessage& msg)
 {
 #if HAL_LOGGING_ENABLED
-    if (AP::can().get_log_level() != AP_CANManager::LOG_NONE) {
+#if AP_HAVE_GCS_SEND_TEXT
```

Comment:
This shouldn't be nested within `HAL_LOGGING_ENABLED`

Reply:
why not?